modules/apps/style-book/style-book-service/service.xml

```diff
@@ -58,5 +58,6 @@
 		<exception>DuplicateStyleBookEntryKey</exception>
 		<exception>StyleBookEntryFile</exception>
 		<exception>StyleBookEntryName</exception>
+		<exception>ThemeIdNotFoundStyleBookEntryFile</exception>
```

Comment:
There aren't much exceptions named `*NotFound*Exception`. My proposal is to call this exception `StyleBookEntryThemeIdException`, then have an internal exception to handle null values. As an example see [PLOEntryValueException.java](https://github.com/liferay/liferay-portal/blob/0033bed5d9f9563b9bce8aef4de068b94cc32e12/modules/apps/portal-language/portal-language-override-api/src/main/java/com/liferay/portal/language/override/exception/PLOEntryValueException.java#L15).

Also, this exception name is wrong, it's not that the themeId wasn't found, but it is not present

Reply:
There are some, but it seems most of the exceptions which extends `PortalException` does not follow this pattern :thinking: 

Also, the context of `themeId` existence here is related to the file, not db. "Not found" and "Not present" means the same :thinking:

What do you think? @ethib137